src/customJs/socialTool.js

```diff
@@ -1,6 +1,8 @@
+// eslint-disable-next-line no-undef
```

Comment:
¿Por qué estamos agregando reglas de omisión a los archivos? No es buena práctica cuando no es estrictamente necesario.

Reply:
Lo que sucede con estos archivos es que no hacen parte del ecosistema de React por eso tenemos tantos errores de eslint y prettier, le agregué esas líneas para deshabilitarlo en esta tarea podremos solucionar esta configuración https://makingsense.atlassian.net/browse/DE-271

Reply:
Pienso que sería lo mejor, porque es un problema nuevo que se sale del scope de esta tarea.

Reply:
Me parece bien la solución de @Cromeror, pero también es buena la explicación para aclarar la duda de @paangaflo que puede ser la de cualquiera de nosotros la semana que viene (uno se olvida...).

Que opinás de poner un comentario con una aclaración?, algo como lo siguiente:

```suggestion
// TODO: remove these exceptions
// These rules are disabled by the moment because we are using this file out of the react context. 
// See https://makingsense.atlassian.net/browse/DE-271
// eslint-disable-next-line no-undef
```

Reply:
Uh, no había visto que son tantas las entradas, no tiene sentido poner tantos TODOs 😛

Creo que están bien como un paso temporal.

Reply:
Si es transitorio, eso lo resolveremos en esta tarea https://makingsense.atlassian.net/browse/DE-271

Reply:
Procedo a resolver este hilo para continuarlo en la tarea creada para solucionar esto.